.github/workflows/prbot.yml

```diff
@@ -1,18 +1,90 @@
 name: PR Robot
-on:
-- pull_request
-
+on: [pull_request]
 jobs:
-  pr-robot:
+  report-size:
```

Comment:
The report-size job takes a long time, since it builds both the current commit and `master` from scratch. I could try to optimize it to use both the cache from `master` and the cache from the current commit, but since this job runs in parallel with the entire rest of the CI, this may not save time overall.

Reply:
I think we can speed this report with esbuild. I want to revisit the bundle size report, perhaps just comparing the browser esm files which we can generate quickly. Let's not optimize for now,